.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.pylintEnabled": true
+}
```

Comment:
Please, keep your environment-specific files out of the repository. You can use a global `.gitignore` in your system or `.git/info/exclude` for this specific project.